datafusion/proto/src/logical_plan/mod.rs

```diff
@@ -1386,6 +1388,7 @@ impl AsLogicalPlan for LogicalPlanNode {
                     options,
                     constraints,
                     column_defaults,
+                    ..
```

Comment:
here i am undecided about how to handle the flag, if so it would have to be serialised with `#[prost]`?

Reply:
We should add a `temporary` flag to the end of  [CreateExternalTableNode](https://github.com/apache/datafusion/blob/da70fab0dd295bd6e00b872dd47e10a877d5ddf4/datafusion/proto/proto/datafusion.proto#L158) and [CreateView](https://github.com/apache/datafusion/blob/da70fab0dd295bd6e00b872dd47e10a877d5ddf4/datafusion/proto/proto/datafusion.proto#L198) in datafusion.proto.
Then Run the `regen.sh`.